### DISCUSSION TITLE
課題04のコンソール出力ができないことについて

### DISCUSSION THREAD

**Discussion by TakedaAyano** [nsc-kiwamu/traning-takeda#62] [Q&A]
課題04のコンソールを確認していたのですが、解決策が見つかりませんでした。
４７行目くらいから、console.log(Fruit1)と書き込み上から実行していくと出力されたのですが、58行目以降に書き込むとなぜかコンソールで出力されなくなってしまいます・・。

**Comment by nsc-kiwamu** [comment]
for文の終了条件を見直してみて下さい
いわゆる「無限ループ」になっています。
